src/main/java/com/google/cloud/spanner/jdbc/JdbcDataType.java

```diff
@@ -229,6 +229,32 @@ public Type getSpannerType() {
       return Type.string();
     }
   },
+  JSON {
+    @Override
+    public int getSqlType() {
+      return JsonType.VENDOR_TYPE_NUMBER;
```

Comment:
JDBC does not have anything like a JSON type, and we need to have a specific type code for this in order to be able to set JSON values as parameters on `PreparedStatement`. Note that this is also what is needed when working with for example PostgreSQL through JDBC: https://stackoverflow.com/questions/35844138/how-can-i-insert-json-object-into-postgres-using-java-preparedstatement